axelor-stock/src/main/resources/domains/StockMove.xml

```diff
@@ -76,6 +76,8 @@
     <string name="origin" readonly="true" title="Origin"/>
 
     <enum name="modeOfTransport" ref="com.axelor.apps.stock.db.ModeOfTransport"/>
+    
+    <string name="saleOrderSpecificPackage" large="true"/>
```

Comment:
add field on supplychain module only and display only if supplychain app is  installed